src/main/java/uk/gov/hmcts/reform/orgrolemapping/data/BatchLastRunTimestampEntity.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.orgrolemapping.data;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.Entity;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Id;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import java.util.Date;
+
+@Builder(toBuilder = true)
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@SequenceGenerator(name = "batch_last_run_timestamp_id_seq", sequenceName = "batch_last_run_timestamp_id_seq",
+        allocationSize = 1)
+@Entity(name = "batch_last_run_timestamp")
+public class BatchLastRunTimestampEntity  {
+
+    @Id
+    @Column(name = "id", updatable = false)
+    @GeneratedValue(strategy = GenerationType.IDENTITY, generator = "batch_last_run_timestamp_id_seq")
+    private String batchLastRunTimestampId;
```

Comment:
in your sql, your using bigserial but here we have the datatype as String. i think this may cause issues. should we use Long ?